src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -141,26 +144,50 @@ public bool IsAttributeDerivedFromClass(ITypeSymbol attributeType, ITypeSymbol t
 		/// <summary>
 		/// Query if Acumatica attribute is bound.
 		/// </summary>
-		/// <param name="attributeType">Type of the attribute.</param>
+		/// <param name="attribute">Data of the attribute.</param>
 		/// <returns/>
-		public bool IsBoundAttribute(ITypeSymbol attributeType)
+		public BoundType IsBoundAttribute(AttributeData attribute)
 		{
-			attributeType.ThrowOnNull(nameof(attributeType));
+			attribute.ThrowOnNull(nameof(attribute));
 
-			if (!attributeType.InheritsFromOrEquals(_eventSubscriberAttribute))
-				return false;
+			if (!attribute.AttributeClass.InheritsFromOrEquals(_eventSubscriberAttribute))
+				return BoundType.Unknown;
 
-			return BoundBaseTypes.Any(boundBaseType => IsAttributeDerivedFromClassInternal(attributeType, boundBaseType));
+			if (BoundBaseTypes.Any(boundBaseType => IsAttributeDerivedFromClassInternal(attribute.AttributeClass, boundBaseType)))
+				return BoundType.DbBound;
+
+			if (attribute.AttributeClass.GetMembers().Select(a => a.Name).Contains(IsDBField))
```

Comment:
@saloponov @dropsonic  should there be case sensitive check? Usually we use case insensitive checks

Reply:
@dropsonic @saloponov I think this check should be implemented differently:
```
bool containsIsDbFieldproperty = 
	attribute.AttributeClass.GetMembers()
							.OfType<IPropertySymbol>()
							.Any(property => IsDBField.Equals(property.Name, StringComparison.OrdinalIgnoreCase));

if (containsIsDbFieldproperty)
{
	var isDbPropertyAttributeArgs = attribute.NamedArguments
											 .Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase))
											 .ToList();

	if (isDbPropertyAttributeArgs.Count != 1)
		return BoundType.Unknown;

	if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
		return BoundType.Unknown;

	return isDbPropertyAttributeArgument
		? BoundType.DbBound
		: BoundType.Unbound;
}
```